packages/cli/src/main_parser_factory.ts

```diff
@@ -12,6 +12,7 @@ import * as path from 'path';
 export const createMainParser = (libraryVersion: string): Argv => {
   const parser = yargs()
     .version(libraryVersion)
+    .alias('v', 'version')
```

Comment:
nit: mind grouping `.alias` calls together ?

Reply:
grouped the alias calls